torch/csrc/autograd/python_variable_indexing.cpp

```diff
@@ -86,6 +86,9 @@ static inline Variable valueToTensor(c10::TensorOptions options, PyObject* value
   if (PyFloat_Check(value)) {
     return at::indexing::scalarToTensor(Scalar(THPUtils_unpackDouble(value)), options, device);
   }
+  if (PyComplex_Check(value)) {
```

Comment:
Why is this needed now and was not before?

Reply:
this is used in the assignment of complex scalar to a variable in `get_numerical_jacobian` method [here](https://github.com/pytorch/pytorch/blob/3799d1d74a39cd3d4419539453acc7f5f7c63f7e/torch/autograd/gradcheck.py#L131-L136)